packages/tools/examples/CINETool/index.ts

```diff
@@ -147,31 +212,97 @@ async function run() {
-    type: ViewportType.STACK,
-    element,
-    defaultOptions: {
-      background: <Types.Point3>[0.2, 0, 0.2],
+  const viewportIds = [
+    'CT_AXIAL_STACK',
+    'CT_CORONAL_VOLUME',
+    'CT_SAGITTAL_VOLUME',
+    'CT_OBLIQUE_VOLUME',
+  ];
+
+  const viewportInputArray = [
+    {
+      viewportId: viewportIds[0],
+      type: ViewportType.STACK,
+      element: elements[0],
+      defaultOptions: {
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
     },
-  };
+    {
+      viewportId: viewportIds[1],
+      type: ViewportType.ORTHOGRAPHIC,
+      element: elements[1],
+      defaultOptions: {
+        orientation: Enums.OrientationAxis.CORONAL,
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+    {
+      viewportId: viewportIds[2],
+      type: ViewportType.ORTHOGRAPHIC,
+      element: elements[2],
+      defaultOptions: {
+        orientation: Enums.OrientationAxis.SAGITTAL,
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+    {
+      viewportId: viewportIds[3],
+      type: ViewportType.ORTHOGRAPHIC,
+      element: elements[3],
+      defaultOptions: {
+        orientation: {
+          // Random oblique orientation
+          viewUp: <Types.Point3>[
+            -0.5962687530844388, 0.5453181550345819, -0.5891448751239446,
+          ],
+          viewPlaneNormal: <Types.Point3>[
+            -0.5962687530844388, 0.5453181550345819, -0.5891448751239446,
+          ],
+        },
+        background: <Types.Point3>[0.2, 0, 0.2],
+      },
+    },
+  ];
 
-  renderingEngine.enableElement(viewportInput);
+  renderingEngine.setViewports(viewportInputArray);
 
   // Set the tool group on the viewport
-  toolGroup.addViewport(viewportId, renderingEngineId);
-
-  // Get the stack viewport that was created
-  const viewport = <Types.IStackViewport>(
-    renderingEngine.getViewport(viewportId)
+  viewportIds.forEach((viewportId) =>
+    toolGroup.addViewport(viewportId, renderingEngineId)
   );
 
-  // Set the stack on the viewport
-  viewport.setStack(imageIds);
+  // Define a unique id for the volume
+  const volumeName = 'CT_VOLUME_ID'; // Id of the volume less loader prefix
+  const volumeLoaderScheme = 'cornerstoneStreamingImageVolume'; // Loader id which defines which volume loader to use
+  const volumeId = `${volumeLoaderScheme}:${volumeName}`; // VolumeId with loader id + volume id
+
+  // Define a volume in memory
+  const volume = await volumeLoader.createAndCacheVolume(volumeId, {
+    imageIds,
+  });
+
+  volume.load();
+
+  for (const viewportInput of viewportInputArray) {
+    const { viewportId, type } = viewportInput;
+
+    if (type === ViewportType.STACK) {
+      const viewport = <Types.IStackViewport>(
+        renderingEngine.getViewport(viewportId)
+      );
+      viewport.setStack(imageIds);
+      viewport.render();
+    } else if (type === ViewportType.ORTHOGRAPHIC) {
+      setVolumesForViewports(renderingEngine, [{ volumeId }], [viewportId]);
```

Comment:
we only need to setVolumesForVIewports once, you don't need to do it 3 times, since it is [viewportIds]